js/theme.js

```diff
@@ -238,3 +238,33 @@ if (typeof(window) != 'undefined') {
             clearTimeout(id);
         };
 }());
+
+// function for opening/closing sidebar on
+// swiping right/close.
+(function () {
+    var start = null;
+      window.addEventListener("touchstart",function(event){
+          if(event.touches.length === 1){
+              start = event.touches.item(0).clientX;
+          } else {
+              start = null;
+          }
+      });
+
+      window.addEventListener("touchend",function(event){
+          var offset = 100;
+          if(start) {
+              var end = event.changedTouches.item(0).clientX;
+              if(end > start + offset) {
```

Comment:
I'm personally not in favor of expanding the menu using swipes. It is too easy to trigger when using the page (e.g. selecting text). Closing it by swiping is OK